servlet/6.1/_index.md

```diff
@@ -29,7 +29,7 @@ The release removes references to the SecurityManager and provides various small
 * [Jakarta Servlet 6.1 Specification Document](./jakarta-servlet-spec-6.1.pdf) (PDF)
 * [Jakarta Servlet 6.1 Specification Document](./jakarta-servlet-spec-6.1.html) (HTML)
 * [Jakarta Servlet 6.1 Javadoc](./apidocs)
-* [Jakarta Servlet 6.1.0 TCK](https://download.eclipse.org/jakartaee/servlet/6.1/jakarta-servlet-tck-6.1.0.zip), ([sig](https://download.eclipse.org/jakartaee/servlet/6.1/jakarta-servlet-tck-6.1.0.zip.sig), [sha](https://download.eclipse.org/jakartaee/servlet/6.1/jakarta-servlet-tck-6.1.0.zip.sha256), [pub](https://jakarta.ee/specifications/jakartaee-spec-committee.pub))
+* [Jakarta Servlet 6.1.1 TCK](https://download.eclipse.org/jakartaee/servlet/6.1/jakarta-servlet-tck-6.1.1.zip), ([sig](https://download.eclipse.org/jakartaee/servlet/6.1/jakarta-servlet-tck-6.1.1.zip.sig), [sha](https://download.eclipse.org/jakartaee/servlet/6.1/jakarta-servlet-tck-6.1.1.zip.sha256), [pub](https://jakarta.ee/specifications/jakartaee-spec-committee.pub))
```

Comment:
Please do keep the 6.1.0 TCK entry similar to what we did for https://jakarta.ee/specifications/servlet/6.0.

Source for the 6.0 TCK service releases is at https://github.com/jakartaee/specifications/blob/master/servlet/6.0/_index.md?plain=1

The entry should link to the TCK challenge issue(s) that are addressed by the 6.1.1 TCK service release and include a brief description of the accepted issue.